testdata/ede_caching_subnetcache.tdir/ede_caching_subnetcache.pre

```diff
@@ -0,0 +1,43 @@
+# #-- ede_caching_cachedb.pre --#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# use .tpkg.var.test for in test variable passing
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+
+PRE="../.."
+. ../common.sh
+
+$PRE/unbound -V > subnet_module_check.txt
+
+if ! grep -q -e "enable-subnet" subnet_module_check.txt
+then
+	echo "Missing subnet_module in the ./configure stage:"
+	echo "'unbound -V' output:"
+	cat subnet_module_check.txt
+	exit 1
```

Comment:
The skip_test function can be used here.
```suggestion
	skip_test "missing cachedb module in the ./configure stage"
```

Reply:
These tests no longer exist; mark as resolved.